tox.ini

```diff
@@ -5,7 +5,7 @@ envlist = py36
 [testenv]
 deps =
     binaryornot
-    flake8==2.5.5
+    flake8==3.5.0
```

Comment:
I suggest that we address that with a straightforward

```ini
deps = -rrequirements.txt
```

therefore closing #1494.